### PR TITLE
add disconnect method

### PR DESCRIPTION
**Pull request by benoitvidis**
refers to https://github.com/kuzzleio/kuzzle/issues/454

+ added a light docker-compose env to test local plugin (needs to be restarted on code update though)

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -205,6 +205,8 @@ function WebsocketProtocol () {
       }
     }
   };
+
+  this.disconnect = this.onClientDisconnection;
```
Comment (scottinet) [inline lib/index.js]: `onClientDisconnection` cleans up the connections pool after a socket is closed. This means that `disconnect` keeps the client socket open, while it should be closed with an appropriate error code.

File: test/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+version: '2'
```
Comment (ballinette) [inline test/docker-compose.yml]: why do we need this ?
As we are doing only unit-tests in plugins, we shouldn't need to launch a full kuzzle instance to run the tests.

Comment (benoitvidis) [inline test/docker-compose.yml]: I find it handy to be able to functionally test a plugin. 
By testing, I mean playing with it using a sdk, not a real functional test suite (at least for the time being).

Comment (ballinette) [inline test/docker-compose.yml]: Hmmm at least this should be well documented.
(a 2-line paragraph in the Readme could be sufficient imho)

Comment (benoitvidis) [inline test/docker-compose.yml]: Why would we? 
We never document the development tools in the Readme on any other project.

Comment (benoitvidis) [inline test/docker-compose.yml]: I am sorry, but I don't get it and I don't understand why this is blocking the merge.

To me, the readme is absolutely not meant to describe the purpose of every file. It is just there to explain what the project is about and possibly how to contribute and targets external visitors.

If I were seeing this kind of thing in some other project, I would find it... awkward.

Comment (ballinette) [inline test/docker-compose.yml]: My comment was not to block the merge.
But just understand what this file has to do with the PR's purpose.
The PR is "add disconnect method", and the docker-compose file is not needed for that.

And my latest suggestion was only to add a comment **within** the file, at the beginning not in the Readme ;)

But I agree to not block a so little PR for that, so OK for me to merge.

**Pull request merged**